src/vs/workbench/contrib/extensions/test/electron-browser/extensionsWorkbenchService.test.ts

```diff
@@ -223,8 +223,8 @@ suite('ExtensionsWorkbenchServiceTest', () => {
 		assert.equal('1.1.0', actual.version);
 		assert.equal('1.1.0', actual.latestVersion);
 		assert.equal('localDescription1', actual.description);
-		assert.equal('file:///localPath1/localIcon1', actual.iconUrl);
-		assert.equal('file:///localPath1/localIcon1', actual.iconUrlFallback);
+		assert.ok(actual.iconUrl === 'file:///localPath1/localIcon1' || actual.iconUrl === 'vscode-file:/localPath1/localIcon1');
+		assert.ok(actual.iconUrlFallback === 'file:///localPath1/localIcon1' || actual.iconUrlFallback === 'vscode-file:/localPath1/localIcon1');
```

Comment:
where does this url get resolved from ? I would expect the url to be of the form `vscode-file://localhost/localPath1/localIcon1` , seems like I am missing a location for refactor.